doc/source/whatsnew/v0.19.0.txt

```diff
@@ -361,6 +361,8 @@ Google BigQuery Enhancements
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- The ``.get_credentials()`` method of ``GbqConnector`` can now first try to fetch [the application default credentials](https://developers.google.com/identity/protocols/application-default-credentials). This is only valid if the parameter ``private_key`` is not provided. Using this functionality, the ``OAuth2WebServerFlow`` call can be skipped (:issue:`13577`).
```

Comment:
@jreback I can make it shorter like this,

The ``.get_credentials()`` method of ``GbqConnector`` can now first try to fetch [the application default credentials](https://developers.google.com/identity/protocols/application-default-credentials). This is only valid if the parameter ``private_key`` is not provided (:issue:`13577`).

Does this look ok? I didn't understand the other part though - "...and put a ref to the docs you added"
Can you please elaborate. Thanks!